### DISCUSSION TITLE
Beginner, am i doing something wrong or is mod1 key not working?

### DISCUSSION THREAD

**Discussion by dotcorkins** [i3/i3#5891] [Q&A] [answered]
Hello, I just booted the live demo of manjaro i3 off of a USB thumb drive, and i quickly checked the i3 docs, but when i press Mod1 (Alt) + Enter or another key, nothing happens? am i doing something wrong or what?

**Comment by orestisfl** [comment]
We will need your config file to be able to tell what's happening

**Reply by dotcorkins** [reply]
how would i provide this config file?

**Reply by dotcorkins** [reply, marked as answer]
My dumba** didn't realize "superkey" meant the windows key.
